features/Testing.feature

```diff
@@ -3,7 +3,31 @@ Feature: Testing
   I want a Testing.feature file
   So that I can test the cucumber-js-task
 
+  @dorun
   Scenario: A test scenario
     Given I have the number 1 and 3
     When I add them together
-    Then I should have 4
\ No newline at end of file
+    Then I should have 4
```

Comment:
These new tests are here to highlight how this patch affects what tests are being omitted and what are executed